### ISSUE TITLE
Trying to add tracking on Anilist always results in HTTP 429 error

### ISSUE THREAD

**Issue by Kryomancer** [mihonapp/mihon#597]
### Steps to reproduce

1. Log in to Anilist for tracking
2. Click tracking on any manga entry and choose "add tracking"

### Expected behavior

After clicking "add tracking", Mihon should fetch a list of similarly named entries from Anilist and let you choose to track one of them.

### Actual behavior

Only a HTTP 429 error is displayed.

https://github.com/mihonapp/mihon/assets/53344902/97f786cc-0c79-4097-bd2c-686265b611f9

### Mihon version

Stable 0.16.4 (26.2.2024 18:26)

### Android version

Android 13

### Device

Asus Zenfone 8

### Other details

Some people with a similar issue have been suggested that it may be due to having a lot of backed up tracking updates. This is not the case here, as tracking updates go through without issue, this has been an on-going for an entire month now and the worker log shows there is no backlog left:

```
Id: d28eeb11-5e6b-40c7-82f9-8a389e66e09f
Tags:
 - eu.kanade.tachiyomi.data.backup.create.BackupCreateJob
 - BackupCreator
State: ENQUEUED
Next scheduled run: 29.3.2024 20.45
Attempt #1

Id: 333c9f94-787b-4219-8230-a65f487f3242
Tags:
 - LibraryUpdate-manual
 - eu.kanade.tachiyomi.data.library.LibraryUpdateJob
 - LibraryUpdate
State: SUCCEEDED

Id: 149548f1-1b41-479c-9989-a12bb8d575c8
Tags:
 - eu.kanade.tachiyomi.data.download.DownloadJob
 - Downloader
State: CANCELLED

Id: f8fd32ab-b983-4262-aafa-b7c354ce21ba
Tags:
 - DelayedTrackingUpdate
 - eu.kanade.domain.track.service.DelayedTrackingUpdateJob
State: SUCCEEDED

-
```

This happens regardless of wheter I use wifi or mobile data and Anilist opens without errors in browser, so this is happening solely in Mihon.

This issue is similar to #372 but the close reason for that one does not apply to this.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have gone through the [FAQ](https://mihon.app/docs/faq/general) and [troubleshooting guide](https://mihon.app/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[0.16.4](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by Kryomancer** [comment]
I was suggested to log out and back in to Anilist, this seemingly fixed the issue for a few minutes, after which it immediately persisted, and it's now again stuck permanently 429, so there's still definitely something going wrong on Mihon's side.

**Comment by MajorTanya** [comment]
> I was suggested to log out and back in to Anilist, this seemingly fixed the issue for a few minutes, after which it immediately persisted, and it's now again stuck permanently 429, so there's still definitely something going wrong on Mihon's side.

Could you try this again with the 0.16.5 release?

**Comment by Smol-Ame** [comment]
Closing due to no response & as fixed with #411.

I haven't seen anyone bring up AniList 429 error (besides when their servers go down once or twice) in the past couple months since Mihon 0.16.5

Come to the Mihon Discord support channel for help if you are somehow still experiencing this.
